src/Libraries/Microsoft.AspNetCore.Diagnostics.Middleware/Logging/HttpLoggingTagNames.cs

```diff
@@ -34,12 +34,12 @@ public static class HttpLoggingTagNames
     /// <summary>
     /// HTTP Request Headers prefix.
     /// </summary>
-    public const string RequestHeaderPrefix = "RequestHeader.";
+    public const string RequestHeaderPrefix = "http.request.header.";
```

Comment:
New name [http.request.header.](https://github.com/open-telemetry/semantic-conventions/blob/b127e121b9b458e83a0fee8a32f7eb6721912951/docs/http/http-spans.md#common-attributes)

Reply:
We also need to normalize header names:
![image](https://github.com/dotnet/extensions/assets/6381023/0e1fe5e8-169c-495a-8d11-53ac004a96b6)

Reply:
It's defined in https://github.com/dotnet/extensions/blob/61a28bead089f12396158b52bca00df48847a732/src/Libraries/Microsoft.AspNetCore.Diagnostics.Middleware/Logging/HeaderReader.cs#L40

Reply:
Yeah, at first I just wanted to agree if we are going to follow this conventions, and then make such changes.

Reply:
Probably, we'll also need to apply the request header prefix and normalization in the RequestHeaderLogEnricher:
https://github.com/dotnet/extensions/blob/6fda2c8b8f8bacf74eaafff243a10ea20f849aaa/src/Libraries/Microsoft.AspNetCore.Diagnostics.Middleware/Logging/RequestHeadersLogEnricher.cs#L68

Reply:
If we agree on the change, it should be also applied to HttpClient logging of headers